examples/cpp/flow_control/PublisherApp.cpp

```diff
@@ -66,9 +62,11 @@ PublisherApp::PublisherApp(
     //Register the type
     type_.register_type(participant_);
 
-    // Create fast Publisher, which has no controller of its own.
-    fast_publisher_ = participant_->create_publisher(PUBLISHER_QOS_DEFAULT, nullptr, StatusMask::none());
-    if (fast_publisher_ == nullptr)
+    // Create fast Publisher, which has no controller of its own
+    PublisherQos pub_qos = PUBLISHER_QOS_DEFAULT;
+    participant_->get_default_publisher_qos(pub_qos);
+    publisher_ = participant_->create_publisher(pub_qos, nullptr, StatusMask::none());
+    if (publisher_ == nullptr)
```

Comment:
In general, all the ctor code is very compact. IMO it can use some blank lines to improve readability